_startup/zam.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Zam
+mission: Alléger la charge de préparation par le gouvernement du débat parlementaire.
+owner: DINSIC
+incubator: dinsic
+status: investigation
+start: 2018-03-28
+end:
+link:
+repository:
+stats: false
+contact: melodie.dahi@beta.gouv.fr
+---
+# Le contexte
+
+Pour tout projet de loi, l’examen parlementaire est une phase cruciale. Le texte est discuté, retravaillé et voté afin d’aboutir à une loi conforme aux attentes des représentants des citoyens.
+
+Pour le Gouvernement, cette phase est, dans le cycle de vie du projet de loi, celle qui est la plus dense. Les agents doivent répondre aux amendements, parfois très nombreux, déposés par les parlementaires, dans le délai, parfois très court, qui leur est imparti.
+
+Les équipes chargées de coordonner cette procédure sont donc les garants du respect des délais et des procédures de validation, ainsi que de la complétude du travail réalisé et du dossier qui sera fourni aux ministres pour défendre la position du Gouvernement “au banc”, c’est-à-dire lors du débat parlementaire. C’est la raison pour laquelle cette phase de gestion des amendements est à fort enjeu : en dépend la constitution d'un dossier solide et propre pour le ministre.
+
+Concrètement, le travail du coordonnateur consiste à :
+- rester en veille permanente sur les sites Internet du Parlement pour récupérer tout nouvel amendement déposé ;
+- analyser le contenu de chaque amendement et l’adresser à l’agent compétent pour répondre (le nombre d’agents potentiellement concernés peut parfois monter jusqu’à plusieurs centaines) ;
+- s’assurer que chaque réponse passe toutes les étapes de la chaîne de validation prévue et, le cas échéant, organiser les allers-retours ;
+- insérer chaque amendement et sa réponse dans le dossier du ministre, selon le bon format et en prenant garde à ce que tous les documents nécessaires soient présents et à jour ;
+- réaliser un reporting régulier auprès des instances internes voire interministérielles ;
+... le tout en ayant également à rédiger les réponses aux amendements qui relèvent de sa propre compétence métier.
+
+Pour chaque projet de loi, le processus se répète 4 fois : lors de la première lecture à l’Assemblée, pour la première lecture au Sénat, puis aux nouvelles lectures à l’Assemblée et au Sénat. Il peut également y avoir des amendements à traiter lors de l’examen du projet de texte par les commissions parlementaires avant chaque lecture. 
+Le volume des amendements à traiter peut considérablement jouer sur la complexité et le stress que ce travail implique : s’agissant par exemple du projet de loi de financement de la sécurité sociale pour 2018 (PLFSS), il a fallu traiter 1300 amendements en 3 jours, entre le vendredi soir et le début de la séance de débat parlementaire le mardi.
+
+# La solution informatique
```

Comment:
L'emploi des termes **informatique** ici, **harmoniser** et **unification** plus bas me fait penser à la dialectique solution informatique / solution numérique évoquée notamment par Matti Schneider dans *Modernising the state through State Startup, European Public Mosaic (EPuM), September 2017* [[1]](http://www.gencat.cat/eapc/epum/N2/index.html):

> La transformation numérique est le terme utilisé pour un changement très profond dans la manière de concevoir et livrer des services publics. Il ne cherche pas à opposer le stylo et le papier au clavier et à l’écran, mais plutôt de contraster l’informatisation (automatiser ce qui est déjà normé) avec la numérisation (faire émerger de nouveaux usages).

(traduction de Sylvain Dermy)

Reply:
Pour le terme *informatique*, c'est une erreur de copier-coller de la fiche produit, de ma part. On se disait bien que ça faisait désuet. Je corrige ça. Pour le reste, on va y réfléchir.

Reply:
FWIW je détaille cette distinction fondamentale (avec une visée politique) en français dans [_Les conditions d'adoption du numérique dans le service social_](https://hal.archives-ouvertes.fr/hal-01415747/) 🙂